### ISSUE TITLE
is there pre-trained model to share？

### ISSUE THREAD

**Issue by KissyZhou** [scheckmedia/keras-shufflenet#1]
hi，thx for sharing your work。
 i want to try the shuffle-net on my own dataset(small dataset), so i am searching for pre-trained model. would you kindly enough to share your model file with its acc and configuration (like shuffle 1x g3)??

**Comment by scheckmedia** [comment]
Hi, I've updated the repository with weights for group = 3 but unfortunately I was not able to reproduce the accuracy of the paper. 

Accuracy is 0.52

**Comment by KissyZhou** [comment]
@scheckmedia thx for reply. 
what's your group=3 accuracy? >60%?
